### PR TITLE
Small changes to Iris section 3.

### PR DESCRIPTION
**Pull request by pp-mo**
A few readability and language tweaks.

I _haven't_ changed the "Suggested Activity"s to "Exercise"s as it makes the diffs messy.
We can do that all in one later ?

### CODE REVIEW

File: course_content/iris_course/3.Subcube_Extraction.ipynb [hunk 1]
```diff
@@ -522,7 +527,7 @@
    "source": [
     "----\n",
     "\n",
-    "## 3.4 Exercise 3 <a id='exercise'></a>"
+    "## 3.4 Section Review Exercise <a id='exercise'></a>"
```
Comment (pp-mo) [inline course_content/iris_course/3.Subcube_Extraction.ipynb]: Open to a better name for this ?

File: course_content/iris_course/3.Subcube_Extraction.ipynb [hunk 2]
```diff
@@ -279,12 +283,13 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "def less_than_10(cell):\n",
-    "    \"\"\"Return True for values that are less than 10.\"\"\"\n",
-    "    return cell < 10\n",
+    "def near_357p5(cell):\n",
```
Comment (lbdreyer) [inline course_content/iris_course/3.Subcube_Extraction.ipynb]: Should this not be `near_358p5`?

Comment (pp-mo) [reply course_content/iris_course/3.Subcube_Extraction.ipynb]: On second thoughts, and offline discussion, it's better to *not* discuss equality testing here at all.

File: course_content/iris_course/3.Subcube_Extraction.ipynb (hunk 1 above)
Comment (lbdreyer) [reply course_content/iris_course/3.Subcube_Extraction.ipynb]: I was thinking along the same lines.
I'm happy with this for now. If we come up with a better name we can change it later.

**Pull request merged**